content/blog/ausweispflichten.de.md

```diff
@@ -0,0 +1,26 @@
+
+## Dienste bei denen sich der Nutzer angemeldet hat
+
+Grundsätzlich gilt immer, dass der Anbieter dem Nutzer die Ausübung seiner Rechte nicht unmöglich gemacht werden darf. Der Anbieter kann sich nur dann einer Auskunft verweigern, wenn er glaubhaft macht, dass er nicht in der Lage ist, den Betroffenen zu identifizieren, Art. 12 II 2 DSGVO. Identifizierung meint in diesem Kontext, dass der Anbieter sicherstellt, dass es wirklich der Nutzer ist, der für den entsprechenden Account Datenschutzrechte geltend macht. Es ist nicht nötig, dass der Anbieter den Account einer natürlichen Person zuordnen kann. Konkret reicht es, wenn der Nutzer nachweist, dass ihm der Account gehört,[^1] beispielsweise durch die hinterlegte Mailadresse, Benutzername und Kundennummer. In keinem Fall sollten Passwörter weitergegeben werden.
+
+## Ausweiskopien nur in Ausnahmefällen notwendig
+
+Ausweiskopien sind hauptsächlich in den Fällen erforderlich, in denen der Nutzer Rechte gegen einen Verantwortlichen geltend machen möchte, mit dem er zuvor noch nicht selbst in Kontakt stand. In diesen Fällen hat der Verantwortliche meist keine andere Möglichkeit den Nutzer zu identifizieren. Klassischerweise sind das Situationen in denen die Rechte gegen eine Auskunftei gerichtet sind. Entgegen häufig anderslautender Gerüchte, ist der Ausweisinhaber berechtigt, eine Kopie seines Personalausweises zu erstellen, § 20 II PAuswG. Auch hier sind nur diejenigen Informationen erforderlich, die es dem Anbieter erlauben, die Person, die den Anspruch geltend macht, eindeutig dem vorhandenen Datensatz zuzuordnen. Nicht erforderlich sind in aller Regel Ausweisnummer, Lichtbild, persönliche Merkmale und Staatsangehörigkeit. Der Anbieter darf die Ausweiskopie nur dazu nutzen, die Identität zu überprüfen. Danach muss die Kopie wieder vernichtet werden.[^2] Es empfiehlt sich, die nicht benötigten Ausschnitte des Ausweises bereits beim Scannen abzudecken. So läuft man später nicht Gefahr, dass Daten aus einem retuschierten Bild doch wieder herstellbar sind. Der verantwortliche Datenverarbeiter ist dazu verpflichtet, den Nutzer in der Wahrnehmung seiner Rechte aktiv zu unterstützen. Daraus folgt, dass der Anbieter den Identitätsnachweis nicht unter einem Medienbruch verlangen kann. Handelt es sich um eine internetbasierte Plattform, dürfen auch die Identitätsnachweise online erbracht werden (z.B. per Mail).
```

Comment:
Was mir hier fehlt, ist der klare Hinweis, dass der Verantwortliche nur dann eine Ausweiskopie anfordern darf, wenn "begründete Zweifel" bestehen ([1]). Was genau das heißt, wäre sicher auch spannend auszuführen. Ganz einig sind sich die Behörden da auch nicht ([2]).

[1]: https://www.ldi.nrw.de/mainmenu_Aktuelles/Inhalt/Personalausweis-und-Datenschutz/Datenschutz-und-Personalausweis-2019_07.pdf
[2]: https://www.bfdi.bund.de/SharedDocs/Publikationen/AusweiskopieAuskunftsersuchen.html

Reply:
Bei Auskunfteien sollte als Identifikation in aller Regel ja genügen, dass die Daten an die der Auskunftei bekannten Adresse gesendet werden.